### PR TITLE
Order success message for Store pickup order should be distinct from normal message.

### PR DESCRIPTION
**Pull request by nmalevanec**
### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2685: Order success message for Store pickup order should be distinct from normal message
### Manual testing scenarios (*)

1. see https://github.com/magento/inventory/issues/2685

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/InventoryInStorePickupFrontend/Block/Checkout/Onepage/Success.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Get is order has pick in store delivery method.
+     *
+     * @return bool
+     */
+    public function isOrderStorePickup(): bool
+    {
+        $result = false;
+        $order = $this->_checkoutSession->getLastRealOrder();
+        if ($order->getShippingMethod() === InStorePickup::DELIVERY_METHOD) {
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryInStorePickupFrontend/Block/Checkout/Onepage/Success.php]: Should we use `\Magento\InventoryInStorePickupApi\Api\IsStorePickupOrderInterface` instead?

Comment (nmalevanec) [reply app/code/Magento/InventoryInStorePickupFrontend/Block/Checkout/Onepage/Success.php]: @ishakhsuvarov, unfortunately we do not have such constant in IsStorePickupOrderInterface and only in InStorePickup

**Comment by chalov-anton** [comment]
The Success message is different on PR and contains "_We'll email you an order and **pick-up** confirmation..._" text  
we'll email you_on pr2703

Before the PR, the message did not contain "**_pick-up_**" and contained "_**tracking info**_"  
we'll email you_without PR

**Comment by nmalevanec** [comment]
see https://github.com/magento/inventory/pull/2739

**Pull request closed without merging**